azext_iot/central/_help.py

```diff
@@ -528,10 +686,10 @@ def _load_central_command_help():
             type: command
             short-summary: Run a command on a device and view associated response. Does NOT monitor property updates that the command may perform.
             long-summary: |
-                Note: payload should be nested under "request".
-                i.e. if your device expects the payload in a shape {"key": "value"}
-                payload should be {"request": {"key": "value"}}.
-                --content can also be pointed at a filepath like this (.../path/to/payload.json)
+              Note: payload should be nested under "request".
+              i.e. if your device expects the payload in a shape {"key": "value"}
+              payload should be {"request": {"key": "value"}}.
+              --content can also be pointed at a filepath like this (.../path/to/payload.json)
```

Comment:
This help should be with the params or the help examples, not the long summary

Reply:
move the payload to the help example, content to param help (look at how we do help for `az dt model create`

Reply:
Okay.